dev/com.ibm.ws.concurrent.persistent_fat_configupdate_databasetaskstore/semeruFips140_3CustomProfile.properties

```diff
@@ -0,0 +1,14 @@
+# SHA-1 is required for WebSocket Upgrade requests: https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Sec-WebSocket-Accept
```

Comment:
i think the comments on these aren't correct since it's for websockets and we're adding a constraint for derby monitor